lib/src/utils.ts

```diff
@@ -10,3 +10,22 @@ export const rstrip = (str: string, suffix = ' '): string => {
   }
   return str;
 };
+
+export const calculateSKU = async (serverEndpoint = window.location.host): Promise<number> => {
+  const localUnixTimeBefore = Date.now() / 1000;
+  const response = await fetch(serverEndpoint, {
+    method: 'GET',
+    headers: { 'Access-Control-Expose-Headers': 'Date' },
+  });
+
+  const serverDateString = response.headers.get('Date');
```

Comment:
Can you add a check to see if there is a 5xx? I think 500s are not required to set Date

https://stackoverflow.com/a/25586633